tests/php-compatibility.patch

```diff
@@ -21,7 +21,14 @@ index d1fdd9f..a93a3e1 100644
  tests/input/forbidden-comments.php                    8       0
  tests/input/forbidden-functions.php                   6       0
  tests/input/inline_type_hint_assertions.php           7       0
-@@ -33,15 +34,15 @@ tests/input/superfluous-naming.php                    11      0
+ tests/input/LowCaseTypes.php                          2       0
```

Comment:
I don't understand why were these files added in output if total number of errors was increased by 2 only, which corresponds just to added 1 new file, not other 6 files added to output. Can anybody enlighten me?

Reply:
The only line added was the 27:

```diff
++tests/input/negation-operator.php                     2       0
```

The rest is due to how Git `.patch` is generated :)

Reply:
You're seeing a diff for a patch file, so there's double the context: once in the diff that's stored in the patch file, once when displaying the patch file diff itself. Hence the number of changes here.